docs/code/round_2/pages.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
-import re
 from bok_choy.page_object import PageObject
+import re
```

Comment:
Thanks for the pointer to the style guide - I'll update accordingly. I was trying to match the ordering from a nearby spot where the same code snippet appears in the tutorial doc (see https://github.com/edx/bok-choy/blob/master/docs/code/round_3/pages.py#L3). I'll go ahead and commit an ordering fix to that file, instead.